config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -99,6 +99,49 @@ presubmits:
       testgrid-dashboards: sig-windows-presubmit
       testgrid-tab-name: pull-kubernetes-e2e-capz-windows-serial-slow-hpa
       testgrid-num-columns-recent: '30'
+  - name: pull-kubernetes-e2e-capz-windows-cri-stats-alpha
+    decorate: true
+    always_run: false
+    optional: true
+    run_if_changed: 'azure.*\.go$|.*windows\.go$|test/e2e/windows/.*'
+    path_alias: k8s.io/kubernetes
+    branches:
+      - master
+      - main
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+      preset-capz-windows-2022: "true"
+      preset-capz-containerd-1-7-latest: "true"
+      preset-capz-windows-common-pull: "true"
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-azure
+        base_ref: main
+        path_alias: sigs.k8s.io/cluster-api-provider-azure
+        workdir: true
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230406-23cb1879e3-master
+          command:
+            - runner.sh
+            - ./scripts/ci-conformance.sh
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              cpu: 2
+              memory: "9Gi"
+          env:
+            - name: K8S_FEATURE_GATE
```

Comment:
@jsturtevant - Do we also need to set `NODE_FEATURE_GATE`?

Reply:
I think so based on https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/43ba6c080b8af712e5d4c11d2613c4d6dfa15991#diff-14a4ade77eb791f0267e3d1d9b533211dd9651b9f26bdb2db03a6f741310f494

Reply:
woops good catch I think we only need NODE_FEATURE_GATE in this case as I believe the feature is localized to kubelet.

Reply:
Thanks for confirming @jsturtevant , I wasn't a 100% sure from our [last conversation](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3388#issuecomment-1496432775), will update it!

Reply:
@jsturtevant Looking back at your PR where you added the functionality, I noticed the ENV var we want to use is NODE_FEATURE_GATE(S) but we also have used K8S_FEATURE_GATE instead of K8S_FEATURE_GATES in https://github.com/kubernetes/test-infra/pull/29220,  or it doesn't matter?